src/plugins/commander/commands/uptime.js

```diff
@@ -13,33 +13,30 @@ class RuntimeCommand extends Command {
         ];
     }
 
-    call(message) {
-        const client = bot.client;
-        const time = client.uptime;
-
-        const toTimeString = (time) => {
-            const f = Math.floor,
-                s = f(time / 1000),
-                m = f(s / 60),
-                h = f(m / 60),
-                d = f(h / 24);
-
-            const p = (s) => s > 1 ? 's' : '';
+    toTimeString(time) {
+        const f = Math.floor,
+            s = f(time / 1000),
+            m = f(s / 60),
+            h = f(m / 60),
+            d = f(h / 24);
 
-            const seconds = `${s % 60} second${p(s)}`;
-            const minutes = `${m % 60} minute${p(m)}`;
-            const hours = `${h % 24} hour${p(h)}`;
+        const p = (s) => s > 1 ? 's' : '';
 
-            if (d) {
-                const days = `${d} day${p(d)}`;
+        const seconds = `${s % 60} second${p(s)}`;
+        const minutes = `${m % 60} minute${p(m)}`;
+        const hours = `${h % 24} hour${p(h)}`;
 
-                return `${days}, ${hours}, ${minutes} and ${seconds} `;
-            }
+        if (d) {
+            const days = `${d} day${p(d)}`;
 
-            return `${hours}, ${minutes} and ${seconds}`;
+            return `${days}, ${hours}, ${minutes} and ${seconds} `;
         }
 
-        const uptime = toTimeString(time);
+        return `${hours}, ${minutes} and ${seconds}`;
+    }
+
+    call(message) {
+        const uptime = this.toTimeString(bot.client.uptime);
```

Comment:
`bot` is a field on the command, this will throw at runtime

Reply:
done